section_test.go

```diff
@@ -313,3 +314,36 @@ func TestSection_DeleteKey(t *testing.T) {
 		So(f.Section("").HasKey("NAME"), ShouldBeFalse)
 	})
 }
+
+func TestSection_Alias(t *testing.T) {
+	Convey("Map to section alias", t, func() {
+
+		var sectionValueTests = map[string]string{
+			"section":        "a value",
+			"custom.section": "a value",
```

Comment:
```suggestion
			"section":        "value a",
			"custom.section": "value b",
```

I think the test is more useful when two keys have different values. WDTY?